lib/rosette/core/extractor/repo_config.rb

```diff
@@ -104,16 +104,17 @@ module Core
     class RepoConfig
       include Integrations::Integratable
 
-      attr_reader :name, :repo, :locales, :hooks, :description
       attr_reader :extractor_configs, :serializer_configs
+      attr_reader :name, :rosette_config, :repo, :locales, :hooks, :description
       attr_reader :translation_path_matchers, :placeholder_regexes
 
       # Creates a new repo config object.
       #
       # @param [String] name The name of the repository. Usually matches the
       #   name of the directory on disk, but that's not required.
-      def initialize(name)
+      def initialize(name, rosette_config)
```

Comment:
Here, `rosette_config` is the "global" Rosette config object that defines top-level things like the cache and datastore. You can think of it like the parent of each repo config. It wouldn't make sense therefore to put `name` in `rosette_config` because `name` is specific to each repo.